docs/ref/templates/api.txt

```diff
@@ -146,14 +146,20 @@ what's passed by :class:`~django.template.backends.django.DjangoTemplates`.
 
 .. staticmethod:: Engine.get_default()
 
-    When a Django project configures one and only one
-    :class:`~django.template.backends.django.DjangoTemplates` engine, this
-    method returns the underlying :class:`Engine`. In other circumstances it
-    will raise :exc:`~django.core.exceptions.ImproperlyConfigured`.
+    Returns the the underlying :class:`Engine` from the first configured
+    :class:`~django.template.backends.django.DjangoTemplates` engine. If no
+    engines are configured will raise
+    :exc:`~django.core.exceptions.ImproperlyConfigured`.
```

Comment:
:+1: good catch by the way.
(That change was missing in the original patch, wasn't it?)